app/pods/using-the-map-class/controller.js

```diff
@@ -9,22 +9,18 @@ export default Controller.extend({
   model: null, // Populated from route
   owners: readOnly('model'),
 
-  // ownersById: computed('owners.@each.name', function () {
-  //   return this.get('owners').reduce((result, owner) => {
-  //     result[owner.name] = owner;
-  //     return result;
-  //   }, {});
-  // }),
+  // { 'Bob' : { name: 'Bob'... } }
+  ownersByName: computed('owners.@each.name', function () {
+    return this.get('owners').reduce((newMap, owner) => {
+      newMap.set(owner.name, owner);
+      return newMap;
+    }, new Map());
```

Comment:
Good job!

Please note that another way of creating a `Map` is to map the array into an array of arrays like this:

```js
return new Map(
  this.get('owners').map(o => [o.name, o])
);
```